article/RNH_AR_EG_SSTJ_EP_MMC-2016.md

```diff
@@ -98,6 +99,8 @@ in the equations describing the first proposed algorithm. Firstly, the free-wate
 diffusion-weighted signal formula (original article's Methods subsection
 "FWE-DTI") should be written as:
 
+stuff here
```

Comment:
Would be useful to see the original equation so it's easier to see what is the correct modified version.